### PR TITLE
Local user state

### PR DESCRIPTION
**Pull request by nickqweaver**

### CODE REVIEW

File: declaration.d.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+declare module "*.scss";
+declare module "*.graphql" {
+  const value: string;
+  export default value;
+}
+
+export interface IdGetterObj extends Object {
```
Comment (sawyerb22) [inline declaration.d.ts]: Do we still need this?

File: src/app/Login/Login.tsx [hunk 2]
```diff
@@ -3,14 +3,21 @@ import LoginForm from "./LoginForm";
 import { useQuery } from "@apollo/react-hooks";
 import { Redirect } from "react-router";
 
-import { IS_LOGGED_IN } from "./queries/queries";
+import { IS_LOGGED_IN } from "./graphql/queries";
 
 const Login: FC = () => {
   // Is logged in query will eventually hold global user data
   const { data, error } = useQuery(IS_LOGGED_IN);
+  console.log(data);
```
Comment (sawyerb22) [inline src/app/Login/Login.tsx]: You left a comment

File: declaration.d.ts (hunk 1 above)
Comment (nickqweaver) [reply declaration.d.ts]: We are not using it, I will remove

File: src/app/Login/Login.tsx (hunk 2 above)
Comment (nickqweaver) [reply src/app/Login/Login.tsx]: Removed

**Pull request merged**